hardwareControl/hardwareControl.ino

```diff
@@ -1,220 +1,220 @@
+#include "motor.h"
+#include "headlights.h"
+#include "ldr.h"
+
 // wired connections
 #define HG7881_A_IA 11 // D11 --> Motor A Input A --> MOTOR B +
 #define HG7881_A_IB 12 // D12 --> Motor A Input B --> MOTOR B -
 #define HG7881_B_IA 6 // D6 --> Motor B Input A --> MOTOR B +
 #define HG7881_B_IB 7 // D7 --> Motor B Input B --> MOTOR B -
 #define LEFT_LED 2
 #define RIGHT_LED 4
-#define LDR_1 0
-#define LDR_2 1
-#define LDR_3 2
+#define LDR_0 0
+#define LDR_1 1
+#define LDR_2 2
  
 // functional connections
-#define MOTOR_A_PWM HG7881_A_IA // Motor A PWM Speed
-#define MOTOR_A_DIR HG7881_A_IB // Motor A Direction
-#define MOTOR_B_PWM HG7881_B_IA // Motor B PWM Speed
-#define MOTOR_B_DIR HG7881_B_IB // Motor B Direction
+#define MOTOR_L_PWM HG7881_A_IA // Left Motor PWM Speed
+#define MOTOR_L_DIR HG7881_A_IB // Left Motor A Direction
+#define MOTOR_R_PWM HG7881_B_IA // Left Motor B PWM Speed
+#define MOTOR_R_DIR HG7881_B_IB // Left Motor B Direction
```

Comment:
These two are the Right Motor, right?

Reply:
Whoops yeah I forgot to change the comment lol